addons/l10n_in_ewaybill_irn/models/l10n_in_ewaybill.py

```diff
@@ -12,17 +12,22 @@ class L10nInEwaybill(models.Model):
 
     _inherit = 'l10n.in.ewaybill'
 
-    is_process_through_irn = fields.Boolean(compute='_compute_is_process_through_irn')
+    is_process_through_irn = fields.Boolean(compute='_compute_is_process_through_irn', store=True, readonly=True)
```

Comment:
Why the store True?

Reply:
So, just to make sure the decision as taken when the ewaybill is created...
```suggestion
    is_process_through_irn = fields.Boolean(compute='_compute_is_process_through_irn', store=True)
```

Reply:
@jco-odoo https://github.com/odoo/upgrade/pull/7553#discussion_r2046475429